### ISSUE TITLE
Gif recording doesn't work on macOS

### ISSUE THREAD

**Issue by aurieh** [whats-this/owo.sh#26]
*title*

**Comment by OkashiOdayakana** [comment]
smh

**Comment by aurieh** [comment]
Should be semi-easy to implement

**Comment by OkashiOdayakana** [comment]
Then do it pls

**Comment by aurieh** [comment]
@OkashiOdayakana I don't have a mac to test on

**Comment by ItsPugle** [comment]
@aurieh I do though, and I can tell you that it doesn't work:
![Borken](https://monosnap.com/file/1laFDzOxkAaNXNPo2WiMwNoF4Iw6Am.png)

**Comment by aurieh** [comment]
@ItsPugle Yeah, that's why I've created this issue. It depends on two linux-specific commands that use linux-specific apis, ffmpeg's x11grab and slop's x11 rectangle select.

**Comment by OkashiOdayakana** [comment]
:eyes:
